src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -837,9 +839,21 @@ protected function style($style, $value, $attr = array())
         }
 
         $v = "<span class=sf-dump-{$style}>".preg_replace_callback(static::$controlCharsRx, function ($c) use ($map) {
-            $s = '<span class=sf-dump-default>';
+            $s = $b = '<span class="sf-dump-default';
             $c = $c[$i = 0];
+            if (($ns = isset($this->displayOptions['notSelectableChars'][$c[$i]]))) {
```

Comment:
Extra brackets

Reply:
@nicolas-grekas Fixed both, I did that on purpose since PhpStorm usually complains about not using `((...))` while doing assignment-in-condition.

@fabpot Maybe fabbot can also check for double brackets if Symfony codestyle doesn't use them?